### ISSUE TITLE
Moodboard plugin doesn't support Sketch 47

### ISSUE THREAD

**Issue by dmoyadev** [Reinvently/moodboards-sketch-plugin#3]
Every time I select the images I want and I click on "generate moodboards" it crash and cause Sketch to quit. This is the information:

**Computer**: Macbook Air 13" (17A045)
**MacOS version:** 10.13 High Sierra
**Sketch version:** 47 (45396)
**Plugin version:** 1.2.0
**Crashlog:**
[crashlog.txt](https://github.com/Reinvently/moodboards-sketch-plugin/files/1464515/crashlog.txt)

**Comment by Flanker4** [comment]
Hi @Lynx901,

Thank you for detailed report.
Could you also provide the search query and image which causes the crash ?

**Comment by dmoyadev** [comment]
It doesn't matter what I search nor what images I choose, everytime it crash. Here is an example: 
![crash-video](https://user-images.githubusercontent.com/9676003/32698254-95977d34-c7a2-11e7-9c1a-c87ee493315f.gif)

**Comment by Flanker4** [comment]
Hi @Lynx901,
It seems like the Sketch API was changed  in version Sketch 47 and method 
`[MSImageData initWithImage:convertColorSpace:]:`
was removed from MSImageData
https://github.com/abynim/Sketch-Headers/blob/master/Headers/MSImageData.h

Thank you for reporting the issue. I will release a new version of plugin with fix on Mondey

**Comment by dmoyadev** [comment]
No problem! Thanks for your fast answering and fix  :)

**Comment by Flanker4** [comment]
Hi @Lynx901 
Could you check this version of plugin 
https://github.com/Reinvently/moodboards-sketch-plugin/releases/tag/v1.3.0?

Thank you

**Comment by dmoyadev** [comment]
When I install the version 1.3.0, it says that it was replaced, but it says that version 1.2.0 is replaced by version 1.2.0 which is weird. 

![image](https://user-images.githubusercontent.com/9676003/32719539-14eeb2d2-c861-11e7-9b09-2e177684f714.png)

I tried to do a clean installation but it keeps saying the same, so I don't know exactly if something has changed.

And then, when I click on "generate moodboard" it gets freeze for a second and then Sketch crash and close. Here is the report:

[crashlog 1.3.0.txt](https://github.com/Reinvently/moodboards-sketch-plugin/files/1466313/crashlog.1.3.0.txt)
